src/components/views/rooms/PresenceLabel.tsx

```diff
@@ -44,6 +44,8 @@ export default class PresenceLabel extends React.Component<IProps> {
         // the 'active ago' ends up being 0.
         if (presence && BUSY_PRESENCE_NAME.matches(presence)) return _t("presence|busy");
 
+        if (presence === "unreachable") { return _t("presence|unreachable"); }
```

Comment:
What MSC defines presence=unreachable? Shouldn't it be unstable/vendor prefixed?

Reply:
yes, but it's not yet [prefixed](https://github.com/vector-im/synapse-private/blob/clokep/unreacahable/xxxx-unreachable-presence-state.md). Also my test setup is just using `unreachable` for now.